.vscode/settings.json

```diff
@@ -13,5 +13,17 @@
     "**/Thumbs.db": true,
     "**/node_modules": false
   },
-  "eslint.validate": ["json"]
+  "eslint.validate": ["json"],
+  "cSpell.words": ["mantine"],
```

Comment:
what's cSpell? if its the spell check we can remove this

Reply:
That line is for words I add to the 'dictionary' that Spell Checker didn't recognize. I'll delete it and remove the extension since it wasn't too impactful for me.